src/win/d3d_disp.cpp

```diff
@@ -95,6 +95,7 @@ typedef struct D3D_DISPLAY_PARAMETERS {
    int window_x, window_y;
    /* Not owned. */
    const char *window_title;
+   int exist_window;
```

Comment:
`existing_window` would make it clearer what this is. Could this be HWND as well instead of an int?